openapi/paths/checkout-forms@{id}.yaml

```
@@ -0,0 +1,93 @@
+parameters:
+  - $ref: ../components/parameters/organizationId.yaml
+  - $ref: ../components/parameters/resourceId.yaml
+get:
+  tags:
+    - Checkout Forms
+  summary: Retrieve a checkout form
+  operationId: GetCheckoutForm
+  description: |
+    Retrieve a checkout form with specified identifier string
+  responses:
+    '200':
+      description: Checkout form was retrieved successfully
+      content:
+        application/json:
+          schema:
+            $ref: ../components/schemas/CheckoutForm.yaml
+    '401':
+      $ref: ../components/responses/Unauthorized.yaml
+    '403':
+      $ref: ../components/responses/Forbidden.yaml
+    '404':
+      $ref: ../components/responses/NotFound.yaml
+put:
+  tags:
+    - Checkout Forms
+  summary: Create or update a checkout form
+  operationId: PutCheckoutForm
+  description: |
+    Create or update a checkout form with predefined identifier string
```

```suggestion
    Create or update a checkout form with a predefined identifier string.
```
Remember the operation descriptions also need to end with periods (`.`) because these are sentenced.